transit/script/obs.py

```diff
@@ -0,0 +1,152 @@
+import os
+
+import pandas as pd
+import tomllib
+from transit_function import (
+    dataframe_to_markdown,
+    format_numeric,
+    read_transit_assignments,
+)
+
+
+def process_obs_data(
+    transit_input_dir,
+    markdown_output_dir,
+    observed_MUNI_Line,
+    observed_BART,
+    observed_BART_county,
+    observed_BART_SL,
+    observed_SL,
+    observed_NTD,
+    obs_MUNI_line_md,
+    obs_BART_station_md,
+    obs_BART_county_md,
+    obs_BART_SL_md,
+    obs_Screenlines_md,
+    obs_NTD_md,
+):
+    obs_MUNI_line = pd.read_csv(transit_input_dir / observed_MUNI_Line)
+    obs_MUNI_line["Ridership"] = obs_MUNI_line["Ridership"].apply(
+        lambda x: format_numeric(x)
+    )
+    dataframe_to_markdown(
+        obs_MUNI_line,
+        os.path.join(markdown_output_dir, obs_MUNI_line_md),
```

Comment:
since you're using pathlib Path elsewhere (e.g. line 28), can just stick to that and NOT import os. Just import pathlib as Path and use the `/` syntax